vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverter.java

```diff
@@ -79,6 +86,65 @@ public VariantContext transform(Variant variant) {
         return variantContext;
     }
 
+    private String getAnnotationAttributes(Variant variant) {
+        List<ConsequenceType> consequenceTypes = variant.getAnnotation().getConsequenceTypes();
+        String allele = variant.getAnnotation().getAlternativeAllele();
+        String csq = "";
+        if (consequenceTypes != null) {
+            List<String> consequences = new ArrayList<>();
+            List<String> symbols = new ArrayList<>();
+            List<String> genes = new ArrayList<>();
+            List<String> features = new ArrayList<>();
+            List<String> bioTypes = new ArrayList<>();
+            List<String> cDnaPositions = new ArrayList<>();
+            List<String> cdsPositions = new ArrayList<>();
+
+            for (ConsequenceType consequenceType : consequenceTypes) {
+
+                List<ConsequenceType.ConsequenceTypeEntry> soTerms = consequenceType.getSoTerms();
+                for (ConsequenceType.ConsequenceTypeEntry entry : soTerms) {
+                    if (entry != null) {
+                        consequences.add(entry.getSoName());
+                    }
+                }
+                String symbol = consequenceType.getGeneName();
+                if (symbol != null && !symbol.isEmpty()) {
+                    symbols.add(symbol);
+                }
+                String gene = consequenceType.getEnsemblGeneId();
+                if (gene != null && !gene.isEmpty()) {
+                    genes.add(gene);
+                }
+                String feature = consequenceType.getEnsemblTranscriptId();
+                if (feature != null && !feature.isEmpty()) {
+                    features.add(feature);
+                }
+                String bioType = consequenceType.getBiotype();
+                if (bioType != null && !bioType.isEmpty()) {
+                    bioTypes.add(bioType);
+                }
+                Integer cDnaPosition = consequenceType.getcDnaPosition();
+                if (cDnaPosition != null) {
+                    cDnaPositions.add(cDnaPosition.toString());
+                }
+                Integer cdsPosition = consequenceType.getCdsPosition();
+                if (cdsPosition != null) {
+                    cdsPositions.add(cdsPosition.toString());
+                }
+            }
+            csq = allele + "|"
+                    + String.join(",", consequences) + "|"
```

Comment:
It remains the question, how do we separate the so terms for a given consequence type, it it has several? Not with ",", to preserve VCF format. Can we use & as suggested by Ensembl? Or should we reserve & for the "," returned by VEP, and use a different character, like ";"?